.idea/modules/htsjdk.iml

```diff
@@ -39,5 +39,7 @@
         <SOURCES />
       </library>
     </orderEntry>
+    <orderEntry type="library" name="apache-ant-1.8.2-bzip2" level="application" />
+    <orderEntry type="library" name="testng-5.5-jdk15" level="application" />
```

Comment:
git rm --cached --  .idea/modules/htsjdk.iml
<https://github.com/samtools/htsjdk/pull/361#discussion_r43421100>
git commit -m"removing mistakenly added .iml file"

should do it

Y.

On Thu, Oct 29, 2015 at 1:57 PM, Kylee Bergin <notifications@github.com>
wrote:

> In .idea/modules/htsjdk.iml
> <https://github.com/samtools/htsjdk/pull/361#discussion_r43421100>:
>
> > @@ -39,5 +39,7 @@
> >          <SOURCES />
> >        </library>
> >      </orderEntry>
> > +    <orderEntry type="library" name="apache-ant-1.8.2-bzip2" level="application" />
> > +    <orderEntry type="library" name="testng-5.5-jdk15" level="application" />
>
> Ugh I didn't git add this! I'll need a reminder on how to get this file
> out of my commits
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/samtools/htsjdk/pull/361/files#r43421100>.
>